### PR TITLE
add test

### PR DESCRIPTION
**Pull request by pkdotson**

### CODE REVIEW

File: test1.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const add = require('./algo1.js');
+
+const test = (a,b,c) => {
```
Comment (songz) [inline test1.js]: could you come up with better names for a,b,c?

**Review by allopez7 — COMMENTED** [review]
algo1 is not very descriptive of what you're doing.

**Comment by evo777** [comment]
Maybe you can space your commas and parameters.

**Review by evo777 — APPROVED** [review]
Looks good.

**Pull request merged**